### ISSUE TITLE
Quick shortcut for parser values

### ISSUE THREAD

**Issue by nicolas-zozol** [d-plaindoux/masala-parser#130]
In TS doc, we often write:

 ```js
const parser = C.number().map (n => n*2);
const value = parser.parse(Streams.ofString('1')).value;
test.equal(value, 2 );
```

Adding a `Parser.val(txt:string)` method could reduce boilerplate:

 ```js
const value = C.number().map (n => n*2).val('1');
test.equal(value, 2 );
```
There may be a downside on genlex parsers to be tested before.

**Comment by nicolas-zozol** [comment]
Another option is to overload parse method with `parse(text:string)` that would returns the Response using StringStream

**Comment by nicolas-zozol** [comment]
Closed by 85ecec9dad6100584859146bfbd7712b9150776e
